api-report/quorum.api.md

```diff
@@ -0,0 +1,71 @@
+## API Report File for "@fluid-experimental/quorum"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { IChannelAttributes } from '@fluidframework/datastore-definitions';
+import { IChannelFactory } from '@fluidframework/datastore-definitions';
+import { IChannelStorageService } from '@fluidframework/datastore-definitions';
+import { IFluidDataStoreRuntime } from '@fluidframework/datastore-definitions';
+import { IFluidSerializer } from '@fluidframework/shared-object-base';
+import { ISequencedDocumentMessage } from '@fluidframework/protocol-definitions';
+import { ISharedObject } from '@fluidframework/shared-object-base';
+import { ISharedObjectEvents } from '@fluidframework/shared-object-base';
+import { ISummaryTreeWithStats } from '@fluidframework/runtime-definitions';
+import { SharedObject } from '@fluidframework/shared-object-base';
+
+// @public
+export interface IQuorum extends ISharedObject<IQuorumEvents> {
+    // (undocumented)
+    delete(key: string): void;
+    // (undocumented)
+    get(key: string): any;
+    // (undocumented)
+    getPending(key: string): any;
+    // (undocumented)
+    has(key: string): boolean;
+    // (undocumented)
+    set(key: string, value: any): void;
+}
+
+// @public (undocumented)
+export interface IQuorumEvents extends ISharedObjectEvents {
+    (event: "pending" | "accepted", listener: (key: string) => void): any;
+}
+
+// @public
+export class Quorum extends SharedObject<IQuorumEvents> implements IQuorum {
```

Comment:
Well, we'll need to export some runtime object or else there's no way to instantiate it :-P  Right now the `Quorum` class is the only runtime export -- we could swap this out for only exporting the factory, but it would deviate from what we do for other DDSs (and I think we should be consistent across the board either way).